roles/identity-management/manage-rh-sso-identities/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+# Username must be lowercase
```

Comment:
We should capture this in the README, but also look at using a filter to enforce this in the role)

Reply:
Perfect. I will make sure to structure the README file to contain these instructions.